packages/calypso-e2e/src/lib/pages/plugins-page.ts

```diff
@@ -54,4 +60,36 @@ export class PluginsPage {
 			assert.notEqual( title, section );
 		}
 	}
+
+	/**
+	 * Select monthly
+	 */
+	async selectMonthlyPricing(): Promise< void > {
+		await this.page.click( selectors.pricingToggle );
+		await this.page.waitForSelector( selectors.pricingToggleOpen );
+		await this.page.click( selectors.monthlyPricingSelect );
+	}
+
+	/**
+	 * Select annual
+	 */
+	async selectAnnualPricing(): Promise< void > {
+		await this.page.click( selectors.pricingToggle );
+		await this.page.waitForSelector( selectors.pricingToggleOpen );
```

Comment:
Still something happening here, the toggle gets clicked but then disappears. There is some kind of z-index showing up via `.is-open` but can't seem to target `annualPricingSelect` at this point.


https://user-images.githubusercontent.com/811776/155270229-c41c1b20-378c-4750-9f9f-45da79e73484.mp4

Reply:
Clicking any sub element managed to work, added text selectors instead.